setup.py

```diff
@@ -5,4 +5,5 @@
     version="0.1",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
+    package_data={"delphi.static": ["*"]},
```

Comment:
Does this line mean:
1. There is *package_data* in `delphi/static` and
2. `["*"]` states that everything in `delphi/static` should be considered part of the package_data?